lib/runner/cli/cli.js

```diff
@@ -128,9 +128,27 @@ class CliRunner {
         '  Please consult the docs at: https://nightwatchjs.org/gettingstarted#settings-file');
     }
 
+    this.validateChromeOptions();
+
     return this;
   }
 
+  validateChromeOptions() {
+    Object.keys(this.baseSettings.test_settings).map((testSettingKey) => {
+      let testSetting = this.baseSettings.test_settings[testSettingKey];
+      if (Utils.isObject(testSetting.desiredCapabilities) &&
+        Utils.isObject(testSetting.desiredCapabilities.chromeOptions) &&
+        Utils.isObject(testSetting.desiredCapabilities.chromeOptions.args)) {
+        this.invalidChromeArguments = Utils.findUnquotedCliArgumentsWithWhitespaces(testSetting.desiredCapabilities.chromeOptions.args);
+
+        console.error('Unquoted argument values contains whitespace which may leads to errors : [ ' + this.invalidChromeArguments +
+          ' ]  check your configuration to prevent future issues.');
+
+
```

Comment:
I think you want to remove these extra blank lines.

Reply:
ofc ! done :)